src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,35 @@
+package qna.domain;
+
+import org.hibernate.annotations.Where;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import javax.persistence.OrderBy;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Embeddable
+public class Answers {
+    @OneToMany(mappedBy = "question", cascade = CascadeType.ALL)
+    @Where(clause = "deleted = false")
+    @OrderBy("id ASC")
+    private List<Answer> answers = new ArrayList<>();
+
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+
+    public boolean isOwner(User loginUser) {
+        return answers.stream().noneMatch(answer -> !answer.isOwner(loginUser));
+    }
+
+    public void writeDeleteHistories(List<DeleteHistory> deleteHistories) {
+        for (Answer answer : answers) {
+            answer.setDeleted(true);
```

Comment:
```suggestion
            answer.delete(loginUser);
```

위와 같이 구현하는 것은 어떨까?
setter 메서드 제거

Reply:
넵! 객체에 메시지를 보내는 방식을 생각했어야하는데! 피드백 감사합니다